python_functionality/homework.py

```diff
@@ -14,7 +14,8 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
         fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    l = [{key: (val.title() if key == 'name' else val) for (key, val) in i.items()} for i in data]
+    return l
```

Comment:
:open_mouth:

Reply:
дуже круто

Reply:
але внесу ще свої 5 копійок
```suggestion
    return [{key: (val.title() if key == 'name' else val) for (key, val) in i.items()} for i in data]
```